cl/beacon/handler/pool.go

```diff
@@ -283,28 +283,30 @@ func (a *ApiHandler) PostEthV1ValidatorAggregatesAndProof(w http.ResponseWriter,
 
 	failures := []poolingFailure{}
 	for _, v := range req {
-		if err := a.aggregateAndProofsService.ProcessMessage(r.Context(), nil, v); err != nil && !errors.Is(err, services.ErrIgnore) {
-			log.Warn("[Beacon REST] failed to process bls-change", "err", err)
-			failures = append(failures, poolingFailure{Index: len(failures), Message: err.Error()})
-			continue
-		}
-		// Broadcast to gossip
+		var gossipData *sentinel.GossipData
```

Comment:
use `PeeredObject` here

Reply:
Here I don't even have peer. Btw, I had to add `peer != nil` check.